.github/workflows/support-window.yml

```diff
@@ -1,52 +1,49 @@
 name: Update support window diagram
 on:
-  # When our dependencies change
+  # When our dependencies change.
   push:
     paths:
```

Comment:
Should this also fire if the json file is modified?

Reply:
@jakebailey You're not wrong but I omitted it deliberately because the workflow updates the JSON, like it does the SVG (although only when run manually, as of this PR). It's an artifact vs. a dependency, I think. You _could_ edit the JSON independently, but that's not what I had in mind.